modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/serializeUtil.js

```diff
@@ -413,39 +413,32 @@ function appendXMLNotifications(buffer, notifications, nodeName, exporting) {
 					);
 				});
 			}
-
-			const recipientsAttrs = {};
-
-			if (
-				recipients[index]?.receptionType &&
-				recipients[index]?.receptionType.some(
-					(receptionType) => receptionType !== ''
-				)
-			) {
-				recipientsAttrs.receptionType = recipients[index].receptionType;
-			}
-
-			if (!recipientsAttrs.receptionType && receptionType?.[0]) {
-				recipientsAttrs.receptionType = receptionType[0];
-			}
-
-			recipients[index]?.roleType?.forEach((item, roleTypeIndex) => {
-				if (item === 'depot' || item === 'asset library') {
-					recipients[index].roleType[roleTypeIndex] = roleTypeName;
+			for (let i = 0; i < recipients.length; i++) {
```

Comment:
I need to improve the for logic in case of multiple notifications on the same node